### ISSUE TITLE
Sometimes it uses the goto option without asking (and does it wrongly)

### ISSUE THREAD

**Issue by mratmartinez** [passff/passff#192]
Well, this is actually just one problem. But since I installed this months ago, passff changes its configuration suddenly with no apparent reason.  
I have setted it for just fill on the forms. But, sometimes I use it with Ctrl+Y and Enter to notice that it has disconfigured itself and now it uses the Goto>Fill>Submit.

So, the problem is having to configure it again, and the changes won't work until I restart Firefox. It's so annoying because, another issue that I'm having is that when that happens, it's still not working properly.

For example, I have this password:  
.password-storage/github.com/mratmartinez.gpg

In theory, the Goto option should go to github.com and send a login request. But what it does is: go to http://mratmartinez/

I don't know if that's because a bug or just my password storage structure isn't the right one for using that option. But I wouldn't have noticed this without the first problem.

Also, excuse me if there is any error in my english. If something isn't clear enough, I will try to explain it again differently. I don't think I can provide a screenshot, because as I said, this happens randomly. Most of the times it works right just filling. But some others, it uses the Goto option and tries send me to my nickname instead of the webpage.

**Comment by tuxor1337** [comment]
This is probably a duplicate of #184. Have you tried the latest version from Github?

**Comment by mratmartinez** [comment]
I'm sorry for making a duplicate. I haven't tried with the lastest version from Github, just with the lastest from Firefox.  
By the way, I want to know why the Go-To option works in that way. Thank you for the feedback.

**Comment by nwallace** [comment]
Our recent updates are under review by Mozilla. Once the review is complete, the plugin should be available to update and these problems should be fixed.

>By the way, I want to know why the Go-To option works in that way.

I'm not 100% sure without digging into the code, but I believe the approach is that the "Go-To" option looks for a line in your password file like this:
```
url: https://github.com/login
```
(key names for this line are configurable in your preferences and default to either "url" or "http")

If your password file doesn't have such a line, it assumes that the last part of the filename is the URL, which in your case is incorrect. Does it work if you add the `url` line to your file?
